spec/plaindate.html

```diff
@@ -687,13 +687,11 @@ <h1>ToTemporalDate ( _item_ [ , _options_ ] )</h1>
           1. Let _fieldNames_ be ? CalendarFields(_calendar_, « *"day"*, *"month"*, *"monthCode"*, *"year"* »).
           1. Let _fields_ be ? PrepareTemporalFields(_item_, _fieldNames_, «»).
           1. Return ? DateFromFields(_calendar_, _fields_, _options_).
-        1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+        1. Let _overflow_ be ? ToTemporalOverflow(_options_). <mark>TODO: _overflow_ is unused</mark>.
```

Comment:
It's used in the "_item_ is an Object" branch above. I think what happens in similar cases elsewhere is "Perform ? ToTemporalOverflow(_options_)."